eth/filters/api.go

```diff
@@ -39,6 +40,9 @@ var (
 	errInvalidBlockRange = errors.New("invalid block range params")
 )
 
+// The maximum number of topic criteria allowed
+const maxTopics = int(vm.LOG4 - vm.LOG0)
```

Comment:
What is the relation to the LOG opcodes? 

Unless there's a very strong reason to use the import, I'd prefer to just set the constant without adding it.